lib/std/atomic.zig

```diff
@@ -408,79 +408,87 @@ test spinLoopHint {
     }
 }
 
+pub fn getCacheLineSizeForCpu(cpu: std.Target.Cpu) u16 {
```

Comment:
```suggestion
pub fn cacheLineForCpu(cpu: std.Target.Cpu) u16 {
```

1. If we want to use the term "cache line size", then `cache_line` should also be renamed.
2. I think the `get` prefix only makes sense in the context of a property that's also settable.

Reply:
Considering the doc comment does refer to it as "size of the CPU's cache line", would you prefer that?

Reply:
There's also C++' [`std::hardware_destructive_interference_size`](https://en.cppreference.com/w/cpp/thread/hardware_destructive_interference_size), which seems to be more appropriate considering doc comments like:

```
// x86_64: Starting from Intel's Sandy Bridge, the spatial prefetcher pulls in pairs of 64-byte cache lines at a time.
```

Reply:
> Considering the doc comment does refer to it as "size of the CPU's cache line", would you prefer that?

I'd be fine with that personally. `cache_line` does seem a bit too vague.

> There's also C++' [`std::hardware_destructive_interference_size`](https://en.cppreference.com/w/cpp/thread/hardware_destructive_interference_size), which seems to be more appropriate considering comments like:

On the one hand, this name is technically more correct. On the other hand, I suspect many (most?) people are going to search for "cache line" and won't even think that "destructive interference" has anything to do with what they're looking for... 🤔

Reply:
Maybe searching could be rectified with a good doc comment? The main use case for this is to avoid false sharing, right?

Reply:
There'd also be the option of providing both names, one as an alias of the other - though I'm not sure if that's maybe against `std` style.